grails-app/views/profile/publication.gsp

```diff
@@ -0,0 +1,96 @@
+<%--
+ Created by Temi Varghese on 14/07/15.
```

Comment:
We need to have the same header and breadcrumbs as shown on other pages, so you might need to pull back some more data (e.g. opus title) to set this up.